impl/src/main/java/com/sun/el/parser/ELParserTokenManager.java

```diff
@@ -16,8 +16,6 @@
  
 /* Generated By:JJTree&JavaCC: Do not edit this line. ELParserTokenManager.java */
 package com.sun.el.parser;
-import java.io.StringReader;
-import jakarta.el.ELException;
 
```

Comment:
As per my previous comment (that didn't appear where/how I expected it to) this is a generated class. There isn't much point cleaning it up.